### PR TITLE
apply PSR-2 style rules to NavigationTree

### PR DESCRIPTION
**Pull request by alfredbez**

### CODE REVIEW

**Comment by alfredbez** [comment]
The tests aren't running for some reason: https://travis-ci.org/OXID-eSales/oxideshop_ce/jobs/137457550
This affects not only this build.

**Comment by Sieg** [comment]
Thanks you @alfredbez.

**Comment by MantasVaitkunas** [comment]
Hi, sorry, but we can't accept this pull request, because of backwards compatibility. Method names and class properties names can't be changed.

File: source/Application/Controller/Admin/NavigationTree.php [hunk 1]
```diff
@@ -329,7 +327,7 @@ protected function _checkDemoShopDenials($oDom)
      *
      * @return null
      */
-    protected function removeInvisibleMenuNodes($dom)
+    protected function _removeInvisibleMenuNodes($dom)
```
Comment (MantasVaitkunas) [inline source/Application/Controller/Admin/NavigationTree.php]: Method name shouldn't be changed.

File: source/Application/Controller/Admin/NavigationTree.php [hunk 2]
```diff
@@ -882,21 +877,21 @@ protected function _getDynMenuUrl($iLang, $blLoadDynContents)
...
 
-        return $sLangAbr;
+        return $langAbr;
     }
 
     /**
      * Method is used for overriding.
      */
-    protected function onGettingDomXml()
+    protected function _onGettingDomXml()
```
Comment (MantasVaitkunas) [inline source/Application/Controller/Admin/NavigationTree.php]: Method name shouldn't be changed.

**Comment by alfredbez** [comment]
Should be better now.

**Pull request merged**